pkg/sshd/server_test.go

```diff
@@ -8,6 +10,7 @@ import (
 	"github.com/deis/builder/pkg/controller"
 
 	"github.com/Masterminds/cookoo"
+	"github.com/arschles/assert"
```

Comment:
dont want to be a stickler here but should this be in pkg or can we use antoher matching lib like gomega?